packages/v4/patternfly-docs/components-data.json

```diff
@@ -0,0 +1,354 @@
+  },
+  "text-area": {
+    "illustration": "./images/component-illustrations/text-area.png",
+    "summary": "A <b>text area</b> component is used for entering a paragraph of text that is longer than one line."
+  },
+  "text-input-group": {
+    "illustration": "./images/component-illustrations/text-input-group.png",
+    "summary": "A <b>text input group</b> is a more flexible composable version of a <Link to='/components/text-input'>text input</Link>. It enables consumers of PatternFly to build custom inputs for filtering and similar use cases by placing elements like icons, <Link to='components/chip-group'>chips groups</Link> and <Link to='components/button'>buttons</Link> within a text input."
+  },
+  "text-input": {
+    "illustration": "./images/component-illustrations/text-input.png",
+    "summary": "A <b>text input</b> is used to gather free-form text from a user."
+  },
+  "text": {
+    "illustration": "./images/component-illustrations/text.png",
+    "summary": "A <b>text</b> component can wrap any static HTML content you want to place on your page to provide correct formatting when using standard HTML tags."
+  },
+  "tile": {
+    "illustration": "./images/component-illustrations/tile.png",
+    "summary": "A <b>tile</b> component is a form of selection that can be used in place of a <Link to='/components/radio'>radio button</Link> and is commonly used in <Link to='/components/form'>forms</Link>. A tile appears visually similar to a <Link to='/components/card/'>selectable card</Link>. However, tiles are used specifically when the user is selecting a static option, whereas a selectable card triggers an action or opens a quickstart or sidebar to provide additional information."
+  },
+  "time-picker": {
+    "summary": "A <b>time picker</b> allows users to select a time from a list of options. Time can be displayed in hours and minutes, but not in seconds."
+  },
+  "title": {
+    "summary": "A <b>title</b> component applies top and bottom margins, font-weight, font-size, and line-height to titles. The most common usage for a title is to define headings within a page. For more information about the relationship between title component sizes and HTML heading levels, see the <Link to='/guidelines/typography#customizing-heading-levels'>Typography guidelines</Link>."
+  },
+  "toggle-group": {
+    "illustration": "./images/component-illustrations/toggle-group.png",
+    "summary": "A <b>toggle group</b> is a group of controls that can be used to quickly switch between actions or states."
+  },
+  "toolbar": {
+    "illustration": "./images/component-illustrations/toolbar.png",
+    "summary": "A <b>toolbar</b> allows a user to manage and manipulate a data set. Data can be presented in any valid presentation, a table, a list, or a data visualization (chart), for example. The PatternFly Toolbar component is a flexible layout system that accommodates a variety of configurations that can be adapted to your specific needs."
```

Comment:
Rather than the second sentence here, I think it should say that `the toolbar responsively accommodates controls and displays applied filters in chip groups.` or something like that